lib/criticalrunner.js

```diff
@@ -20,8 +20,9 @@ system.args sent from critical.js:
 		width = system.args[2],
 		height = system.args[3],
 		forceInclude = JSON.parse(system.args[4]),
-		rulesFile = system.args[5],
-		ignoreConsole = system.args.length > 6 && system.args[6] === "--ignoreConsole" ? true : false,
+		initialMap = JSON.parse(system.args[5]),
+		rulesFile = system.args[6],
+		ignoreConsole = system.args.length > 7 && system.args[7] === "--ignoreConsole" ? true : false,
```

Comment:
Similar to the change in `criticalcss.js` this changes the command line signature of this file when it is run with phantom under the assumption that the only dependencies on this file are internal.